generator/uscore/metadata_extractor.rb

```diff
@@ -458,6 +460,11 @@ def add_references(profile_definition, sequence)
         end
       end
 
+      def add_attachments(profile_definition, sequence)
+        attachments = profile_definition['snapshot']['element'].select { |el| el['type'].present? && el['type'].first['code'] == 'Attachment' && el['mustSupport'] }
```

Comment:
I think this needs to check whether `attachment.url` is must support. It is for DocumentReference, but not for DiagnosticReport, so DiagnosticReport could only support inline attachments.

Reply:
It does:

{ |el| el['type'].present? && el['type'].first['code'] == 'Attachment' && el['mustSupport'] }

Reply:
Reverse the changes since attachment.url or attachment.data is must supported.